RecoJets/JetProducers/python/hltMVAJetPuId_cff.py

```diff
@@ -0,0 +1,15 @@
+from RecoJets.JetProducers.hltPUIdAlgo_cff import *
+
+hltMVAJetPuIdCalculator = cms.EDProducer('MVAJetPuId',
```

Comment:
@slava77 here the test we made https://raw.githubusercontent.com/cvernier/cmssw/de813a426f023b19e251dd7a0b8f9259ff0595cc/hlt50ns.py
@silviodonato